lib/Sentry.js

```diff
@@ -245,6 +267,7 @@ class RavenClient {
         this._dsn = dsn;
         this.options = {
             allowSecretKey: true,
+            allowDuplicates: true
```

Comment:
After I thought about it again I will change this to `allowDuplicates` when a native client is available otherwise we keep the default. While I was testing it was strange that only the first event was sent, even if I triggered it multiple times.